Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/distance.h

```diff
@@ -1310,8 +1322,1295 @@ double approximate_symmetric_Hausdorff_distance(const TriangleMesh& tm1,
+    const Point_3 v1 = triangle.vertex(1);
+    const Point_3 v2 = triangle.vertex(2);
+
+    // Recursively process the current triangle to obtain a lower bound on its Hausdorff distance.
+    const FT triangle_bound = recursive_hausdorff_subdivision<Kernel>(
+      v0, v1, v2, tm2_tree, squared_error_bound);
+
+    // Store the largest lower bound.
+    if (triangle_bound > squared_lower_bound) {
+      squared_lower_bound = triangle_bound;
+    }
+  }
+
+  // Return linear interpolation between found upper and lower bound.
+  return CGAL::sqrt(CGAL::to_double(squared_lower_bound));
+}
+
+} // end of namespace internal
+
+/**
+ * \ingroup PMP_distance_grp
+ * returns an estimate on the Hausdorff distance between `tm1` and `tm2` that
+ * is at most `error_bound` away from the actual Hausdorff distance between
+ * the two given meshes.
+ *
+ * @tparam Concurrency_tag enables sequential versus parallel algorithm.
+ *                         Possible values are `Sequential_tag` and `Parallel_tag`.
+ *                         Currently, parallel computation is not implemented, though.
+ *
+ * @tparam TriangleMesh1 a model of the concept `FaceListGraph`
+ * @tparam TriangleMesh2 a model of the concept `FaceListGraph`
+ *
+ * @tparam NamedParameters a sequence of \ref bgl_namedparameters "Named Parameters"
+ *
+ * @param tm1 a triangle mesh
+ * @param tm2 another triangle mesh
+ *
+ * @param error_bound a maximum bound by which the Hausdorff distance estimate is
+ *                    allowed to deviate from the actual Hausdorff distance.
+ *
+ * @param np1 an optional sequence of \ref bgl_namedparameters "Named Parameters" among the ones listed below
+ * @param np2 an optional sequence of \ref bgl_namedparameters "Named Parameters" among the ones listed below
+ *
+ * \cgalNamedParamsBegin
+ *   \cgalParamNBegin{vertex_point_map}
+ *     \cgalParamDescription{a property map associating points to the vertices of `tm1` and `tm2` (`np1` and `np2`, respectively)}
+ *     \cgalParamType{a class model of `ReadablePropertyMap` with `boost::graph_traits<TriangleMeshX>::%vertex_descriptor`
+ *                    as key type and `%Point_3` as value type}
+ *     \cgalParamDefault{`boost::get(CGAL::vertex_point, tm)`}
+ *     \cgalParamExtra{If this parameter is omitted, an internal property map for `CGAL::vertex_point_t`
+ *                     must be available in `TriangleMeshX`.}
+ *   \cgalParamNEnd
+ *   \cgalParamNBegin{match_faces}
+ *     \cgalParamDescription{a boolean tag that turns on the preprocessing step that filters out all faces,
+ *                           which belong to both meshes and hence do not contribute to the final distance}
+ *     \cgalParamType{Boolean}
+ *     \cgalParamDefault{true}
+ *     \cgalParamExtra{Both `np1` and `np2` must have this tag true in order to activate this preprocessing.}
+ *   \cgalParamNEnd
+ * \cgalNamedParamsEnd
+ *
+ * @return the one-sided Hausdorff distance
+ */
+template< class Concurrency_tag,
```

Comment:
shall we remove the tag?

Reply:
I do not think so since it is used in the parallel version.

Reply:
But the conclusion was that parallel does not help for this algorithm, does it?

Reply:
It does not help now, but what if we improve it later? Moreover, the parallel part is not going to compile because it depends on this tag. And we have decided to put that part inside ifdefs and leave it inside the code for the future.

Reply:
@sloriot can we mark this as resolved?

Reply:
you pre-approved I just saw.